### PR TITLE
Bump a bunch of needing-review guidance

### PR DESCRIPTION
**Pull request by timblair**
I also ran the lice site through http://validator.w3.org/checklink and attempted to fix any issues it raised, including updating links that worked but resulted in a permanent redirect.

### CODE REVIEW

File: source/standards/operating-systems.html.md.erb [hunk 1]
```diff
@@ -34,6 +34,6 @@ You must also make sure your OS is:
 * up to date with the latest OS releases
 * able to run on a local VM
 
-The National Cyber Security Centre (NCSC) has principles to follow for [end user device (EUD) hardening](https://www.ncsc.gov.uk/collection/end-user-device-security/eud-overview/eud-security-principles).
+The National Cyber Security Centre (NCSC) has principles to follow for [end user device (EUD) hardening](https://www.ncsc.gov.uk/collection/device-security-guidance/security-principles).
 
-The NCSC also has guidance on [hardening that's specific to `Ubuntu 18.04 LTS`](https://www.ncsc.gov.uk/collection/end-user-device-security/platform-specific-guidance/ubuntu-18-04-lts).
+The NCSC also has guidance on [hardening that's specific to `Ubuntu 18.04 LTS`](https://www.ncsc.gov.uk/collection/device-security-guidance/platform-guides/ubuntu-lts).
```
Comment (lfdebrux) [inline source/standards/operating-systems.html.md.erb:39]: This guidance is now specific to Ubuntu 20.04 LTS, the lint text should be updated, maybe just to:

```suggestion
The NCSC also has guidance on [hardening that's specific to the latest Ubuntu LTS version](https://www.ncsc.gov.uk/collection/device-security-guidance/platform-guides/ubuntu-lts).
```

File: source/standards/principle-least-access.html.md.erb [hunk 2]
```diff
@@ -41,6 +41,6 @@ If you’re using the gds-users account to log into your AWS accounts, you shoul
 
 ## Further Guidance
 
-- [NCSC - 10 steps to cyber security](https://www.ncsc.gov.uk/collection/10-steps-to-cyber-security?curPage=/collection/10-steps-to-cyber-security/the-10-steps/managing-user-privileges)
+- [NCSC - 10 steps to cyber security](https://www.ncsc.gov.uk/collection/10-steps)
 - [NCSC - privileged user management](https://www.ncsc.gov.uk/guidance/introduction-identity-and-access-management#section_6)
 - [NIST Special Publication 800-53 -  AC-6 least privilege](https://nvd.nist.gov/800-53/Rev4/control/AC-6)
```
Comment (lfdebrux) [inline source/standards/principle-least-access.html.md.erb:46]: This link appears to have the same URL as one above which was change; should it also be updated?

**Review by lfdebrux — APPROVED** [review]
Nice one! Looks good to me, just two small snags

File: source/standards/principle-least-access.html.md.erb (hunk 2 above)
Comment (timblair) [reply source/standards/principle-least-access.html.md.erb:46]: Yes! Well spotted.

**Review by lfdebrux — APPROVED** [review]

**Pull request merged**